src/components/admin/lib/__tests__/article-validators.test.js

```diff
@@ -0,0 +1,19 @@
+import { checkForHttp, checkForAbsoluteUrlRegex } from '../article-validators';
+
+describe('Finds http/https in url', () => {
+  it('Returns true if it finds an http (not s) link ', () => {
+    expect(checkForHttp('https://www.google.com')).toEqual(true); // if the function returns true, the url is correct
+    expect(checkForHttp('http://www.google.com')).toEqual(false);
+    expect(checkForHttp('http')).toEqual(false);
+    expect(checkForHttp('https')).toEqual(false); // need to be fixed
+  });
+  it('Returns true if it finds a URL that is in non-absolute format. ', () => {
+    expect(checkForAbsoluteUrlRegex('www.google.com')).toEqual(false);
+    expect(checkForAbsoluteUrlRegex('http://www.google.com')).toEqual(false); // if the function returns true, the url is correct.
+    expect(checkForAbsoluteUrlRegex('<a href="https://www.w3schools.com">Visit W3Schools</a>')).toEqual(true);
```

Comment:
I agree that it's correct that this is the only one that returns true (false), or whatever we define it to be when we find what we want to find, as we're only looking for links, not just urls

Reply:
Do you mean I should change the name of the function?

Reply:
Nope it's just you wrote some comments that you wanted to fix things, I'm just saying that I agree that the function should only recognize this one of all of your test cases.
Actually you should probably have a test case as well inside an `<a>` tag that has a relative url